docs/_config.yml

```diff
@@ -55,7 +55,7 @@ JB :
   #   - Only the following values are falsy: ["", null, false]
   #   - When setting BASE_PATH it must be a valid url.
   #     This means always setting the protocol (http|https) or prefixing with "/"
-  BASE_PATH : false
+  BASE_PATH : /docs/0.6.0-incubating-SNAPSHOT
```

Comment:
There're version number in the doc page header.

https://github.com/Leemoonsoo/incubator-zeppelin/blob/update_install/docs/_includes/themes/zeppelin/_navigation.html#L12

Although it's not using BASE_PATH to display.

both places are documented in docs/README.md to make sure updated when bumping up the version.